index.html

```diff
@@ -1222,6 +1316,27 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents SHOULD re-fetch and process the manifest on every page
+            load. When the processing is successful, user agents MAY apply updated
```

Comment:
The first part is covered by HTML... or at least, we should update the HTML spec instead, in the link relation section.  

```suggestion
            When the processing is successful, user agents MAY apply updated
```

Reply:
Cool, https://html.spec.whatwg.org/#fetching-and-processing-a-resource-from-a-link-element mentions:

the element must delay the load event of the element's node document until all the attempts to fetch and process the linked resource and its critical subresources are complete. 

I can update the language to - On every page load, the manifest is re-fetched and processed. When the processing is successful, ...